### DISCUSSION TITLE
Assigning roles

### DISCUSSION THREAD

**Discussion by chadcancode** [incidentbot/incidentbot#73] [Ideas]
1. It would be nice to not have to require assigning certain roles before resolving an incident. For example, we don't use the communications liason role. That's part of being the incident commander. 
2. When I open an incident, I automatically page an Incident Commander. I would love to auto-assign them to that role.

**Comment by echoboomer** [comment]
@chadcancode I'll look into the logic as to why I was requiring roles to be assigned before resolution. Also, it may be worthwhile to allow enabling/disabling roles too.

**Comment by echoboomer** [comment]
Only incident manager is required to close an incident in the latest release.
